lib/homework1.ex

```diff
@@ -1,18 +1,50 @@
 defmodule Homework1 do
   @moduledoc """
-  Documentation for `Homework1`.
+  This module implements the simple calculation function.
+  You can use the Homework1.calculate/1 function to add, remove, multiply and divide.
```

Comment:
```suggestion
  You can use the `Homework1.calculate/1` function to add, remove, multiply and divide.
```
That way `ex_doc` would _link_ it in the generated documentation automagically.